working-with-your-data/working-with-your-own-data.md

```diff
@@ -47,7 +47,7 @@ If you are retrieving your data from online, perhaps from a publicly available r
 
 **Step 2)** Copy over the [wget template script]({{site.repository_url}}/tree/master/working-with-your-data/template-scripts/wget-TEMPLATE.sh).
```

Comment:
This link appears to be broken. Is this the fix from #12 or has that fix not yet been merged here?

Reply:
I think the link that uses `{{site.repository_url}}` will always be broken in Markdown and I check if something will be broken in the website by serving things locally (these instructions could use some work in the 'when things are already installed' case): https://github.com/AlexsLemonade/training-specific-template#local-development 

It's possible that this was broken because of `tree` instead of `blob`.

Reply:
Changed back and verified it worked (https://github.com/AlexsLemonade/2020-june-training/pull/14/commits/9a35567b378c3346ad67c8cd4d9ff254f286c75f) just in case we are planning on porting these changes to the template repository.